packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -86,6 +115,38 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
     return result;
   }
 
+  /// Stores the engine layer created for this layer in order to reuse engine
+  /// resources across frames for better app performance.
+  ///
+  /// Set this field to the value returned by one of [ui.SceneBuilder]'s "push"
+  /// methods, e.g. [ui.SceneBuilder.pushOpacity].
+  ///
+  /// This value may be passed to [ui.SceneBuilder.addRetained] to communicate
+  /// to the engine that nothing in this layer or any of its descendants
+  /// changed. The native engine could, for example, reuse the texture rendered
+  /// in a previous frame. The web engine could, for example, reuse the HTML
+  /// DOM nodes created for a previous frame.
+  ///
+  /// This value may be passed as `oldLayer` argument to a "push" method to
+  /// communicate to the engine that a layer is updating a previously rendered
+  /// layer. The web engine could, for example, update the properties of
+  /// previously rendered HTML DOM nodes rather than creating new nodes.
+  ui.EngineLayer get engineLayer => _engineLayer;
+  @protected
+  set engineLayer(ui.EngineLayer newLayer) {
```

Comment:
newLayer => value to avoid the confusion over whether it's a Layer or an EngineLayer

Reply:
Done.